roles/lib_utils/src/generate.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+'''
+  Generate the openshift-ansible/roles/lib_openshift_cli/library/ modules.
+'''
+
+import os
+
+# pylint: disable=anomalous-backslash-in-string
+GEN_STR = "#!/usr/bin/env python\n"                                  + \
+          "# pylint: disable=missing-docstring\n"                    + \
+          "#     ___ ___ _  _ ___ ___    _ _____ ___ ___\n"          + \
+          "#    / __| __| \| | __| _ \  /_\_   _| __|   \\\n"        + \
+          "#   | (_ | _|| .` | _||   / / _ \| | | _|| |) |\n"        + \
+          "#    \___|___|_|\_|___|_|_\/_/_\_\_|_|___|___/_ _____\n"  + \
+          "#   |   \ / _ \  | \| |/ _ \_   _| | __|   \_ _|_   _|\n" + \
+          "#   | |) | (_) | | .` | (_) || |   | _|| |) | |  | |\n"   + \
+          "#   |___/ \___/  |_|\_|\___/ |_|   |___|___/___| |_|\n"
+
+OPENSHIFT_ANSIBLE_PATH = os.path.dirname(os.path.realpath(__file__))
+
+FILES = {'yedit.py':  ['doc/license', 'doc/yedit', 'class/base.py', 'class/yedit.py', 'ansible/yedit.py'],
```

Comment:
@kwoodson Totally! That's perfectly reasonable, the explicit vs. implicit decision. :+1: 

Re: `i put them into a dictionary structure`, we're on the same page with this one. It just makes sense to abstract that data from the code :smiley_cat: